pyccel/codegen/printing/cwrappercode.py

```diff
@@ -636,7 +883,13 @@ def _print_Module(self, expr):
             static_funcs = expr.funcs
         function_signatures = '\n'.join('{};'.format(self.function_signature(f)) for f in static_funcs)
 
-        function_defs = '\n\n'.join(self._print(f) for f in expr.funcs)
+        #TODO it should be a better way to do this :
+        interfaces = expr.interfaces
+        interface_funcs = [f.name for i in interfaces for f in i.functions]
+        funcs = interfaces + [f for f in expr.funcs if f.name not in interface_funcs]
```

Comment:
Do you have something in mind for this TODO? I don't see a problem here

Reply:
not anymore.

Reply:
Then the TODO can be removed?

Reply:
yes